### PR TITLE
Revert 3.0 changes in the master branch

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <Description xml:space="preserve">Roslyn.Diagnostics Analyzers</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0.0.0,17.0.0.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0.27130.0,]" />
```
Comment (mavasani) [inline src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest]: This should not be reverted

File: src/PublicApiAnalyzers/Setup/source.extension.vsixmanifest [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <Description xml:space="preserve">Public API Analyzers</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0.0.0,17.0.0.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0.27130.0,]" />
```
Comment (mavasani) [inline src/PublicApiAnalyzers/Setup/source.extension.vsixmanifest]: Same here, all the changes to VSIX manifests should be persisted.

File: src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest (hunk 1 above)
Comment (mavasani) [reply src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest]: Or will this be restored in 3.0.x branch? I am fine with that as well...

Comment (sharwell) [reply src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest]: This has to be reverted, or the 2.9 builds will stop being installable in VS2017.

Comment (sharwell) [reply src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest]: We can address installation in VS2019 as a follow-up. There are multiple options.

Comment (mavasani) [reply src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest]: Ah got you. That makes sense.

File: src/PublicApiAnalyzers/Setup/source.extension.vsixmanifest (hunk 2 above)
Comment (sharwell) [reply src/PublicApiAnalyzers/Setup/source.extension.vsixmanifest]: I'm leaving this as a follow-up per https://github.com/dotnet/roslyn-analyzers/pull/2367#discussion_r276790377.

**Pull request merged**